Build/templates/run_tests.yml

```diff
@@ -55,8 +55,16 @@ steps:
       configuration: $(BuildConfiguration)
       msbuildArguments: '/p:VSTarget=$(VSTarget) /bl:$(Build.SourcesDirectory)\logs\BuildProduct.binlog'
 
+  - task: CacheBeta@2
+    displayName: 'Restore glass binaries from cache'
+    inputs:
+      key: 'glass'
+      path: '$(Build.BinariesDirectory)\raw\binaries'
```

Comment:
This path is not where the glass bits should go?

Reply:
oh. i was trying to cache the output of ''Setup glass test folder''

Reply:
$(Build.SourcesDirectory)   something thing?

Reply:
maybe i need to do this 
```
Copying PythonTests from D:\a\_work\1\s\Python\Tests\GlassTests\PythonTests to D:\a\_work\1\s\GlassTests\PythonTests
Copying PTVS Debugger bits from D:\a\_work\1\b\raw\binaries to D:\a\_work\1\s\GlassTests\Glass

```

Reply:
Setup_glass does a bunch of stuff, but the parts that pull down the glass bits are what you want to cache. Not the rest of it. 

The output folder for that would be `$(Build.SourceDirectory)\GlassTests`, but you'd still have to change setup_glass.py to only do the 'drop' part. Might be worthwhile to just split that part out into a separate script